discord/guild.py

```diff
@@ -750,6 +769,32 @@ def get_member(self, user_id: int, /) -> Optional[Member]:
             The member or ``None`` if not found.
         """
         return self._members.get(user_id)
+    
+    async def getch_member(self, member_id: int) -> Member:
```

Comment:
I get what you are trying to do here, but really? getch?

Wouldn't it be better to just name this get_or_fetch_member()?

I wouldn't know what the hell getch would do without reading the summary...

Reply:
@EQUENOS so what should I do? Rename it to try_member?

Reply:
Thanks for the comment, Dusk

Reply:
recap of what I said in Discord:

To not slam this topic any further than it needs to be:
- ``getch_member()`` is too obscure to return in the documentation.
- New people won't use it, because they don't know it's a concat from get and fetch
- Getch doesn't tell you what it does exactly.
- get_or_fetch_member() covers the load of the method exactly. Method lengths shouldn't be an issue. 
- The Uniqueness of the word getch would make it never be used, because nobody would think about searching for it when they want to get or fetch a member.

Reply:
"getch" is how discord.py's community wanted to name such methods.
`get_or_fetch_member` is an accurate and self-explaining name, but at the same time it's pretty long. I think we can make a poll regarding the name of get-or-fetch methods

Reply:
But actually I think it's okay to name it `get_or_fetch_member`, because it'll probably be called not more than once in a command